src/store/reducers/storage/requestStorageData.ts

```diff
@@ -0,0 +1,21 @@
+import type {AxiosOptions} from '../../../services/api';
+import type {StorageRequestParams} from '../../../types/api/storage';
+
+import {prepareGroupsResponse, prepareStorageResponse} from './utils';
+
+export async function requestStorageData(
+    {
+        version = 'v2',
+        useGroupsHandler,
+        ...params
+    }: StorageRequestParams & {useGroupsHandler?: boolean},
+    options: AxiosOptions,
+) {
+    if (useGroupsHandler && version !== 'v1') {
```

Comment:
useGroupsHandler really looks like name of react hook

was really confused why we pass hooks to api =)

Please change useGroupsHandler to something else >_<